deploy/template.yaml

```diff
@@ -101,7 +101,7 @@ Mappings:
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
       YOTIBASEURL: "https://f2f-yoti-stub-yotistub.review-o.dev.account.gov.uk"
-      YOTISDK: "MockSDK"
```

Comment:
Is the Yotisdk for dev updated? or do we need to revert it back to MockSDK value?